server/grpc_service.go

```diff
@@ -689,6 +728,82 @@ func (s *heartbeatServer) Recv() (*pdpb.RegionHeartbeatRequest, error) {
 	return req, nil
 }
 
+// ReportBuckets implements gRPC PDServer
+func (s *GrpcServer) ReportBuckets(stream pdpb.PD_ReportBucketsServer) error {
```

Comment:
Why not keep the convention to name it `BucketsHeartbeat`?

Reply:
yes, the name called the `BucketsHeartbeat` is better than `ReportBuckets`,  but I think the past internal grpc function is also ok .